### PR TITLE
integrate react library

### PR DESCRIPTION
**Pull request by vermashreyansh05**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -82,6 +82,26 @@ GEM
     ast (2.4.2)
     autoprefixer-rails (10.4.2.0)
       execjs (~> 2)
+    aws-eventstream (1.2.0)
+    aws-partitions (1.678.0)
```
Comment (ror-uday) [inline Gemfile.lock:86]: This file is not needed , remove it this file

File: app/javascript/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const App = () => {
+	
+  return (
+	<>
+	<div>App</div>
+	</>
```
Comment (ror-uday) [inline app/javascript/components/App.jsx:8]: have you run rubocop ?

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -15,15 +15,15 @@ Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
 
-document.addEventListener("turbolinks:load", () => {
+// document.addEventListener("turbolinks:load", () => {
     // Both of these are from the Bootstrap 5 docs
     // var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'))
     // var tooltipList = tooltipTriggerList.map(function(tooltipTriggerEl) {
     //     return new Tooltip(tooltipTriggerEl)
     // })
```
Comment (ror-uday) [inline app/javascript/packs/application.js:24]: remove it commented code

File: app/javascript/packs/hello_react.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Hello.defaultProps = {
+  name: 'David'
+}
+
+Hello.propTypes = {
+  name: PropTypes.string
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  ReactDOM.render(
```
Comment (ror-uday) [inline app/javascript/packs/hello_react.jsx:22]: have you run rubocop?

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<head>
+<title>Ems</title>
+<%= csrf_meta_tags %>
+<%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+<%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (ror-uday) [inline app/views/layouts/application.html.erb:9]: make a proper indentation

File: config/routes.rb [hunk 6]
```diff
@@ -81,6 +81,7 @@
           put 'update_password'
         end
       end
+      get '/dashboard', to: 'employees#dashboard'
```
Comment (ror-uday) [inline config/routes.rb:84]: run rubocop in this file

**Comment by vermashreyansh05** [comment]
It is the code for react integration in the rails program but it doesn't consist of ant react functionality

**Pull request closed without merging**